include/NovelRT/Utilities/Misc.h

```diff
@@ -11,7 +11,7 @@
 #if defined(NDEBUG)
 #define unused(x) (void)(x)
 #else
-#define unused(x) (void)(0)
+#define unused(x) (void)(x)
```

Comment:
So if we're using the same definition in both branches here, why do we need the branch in the first place?

Reply:
I have no idea, I don't even remember making this change.